### DISCUSSION TITLE
Build process

### DISCUSSION THREAD

**Discussion by dgorti** [vakata/jstree#2481] [General]
How do I build JSTree from source? I tried looking at the docs etc. I am trying to make some changes for A11Y compliance.

**Comment by vakata** [comment]
It is a grunt build procedure - so install grunt-cli and run grunt inside the source folder.

**Reply by dgorti** [reply]
It has bower files and it seems they are not required. I was able to build the code after installing grunt. Npm complains about a bunch of deprecated packages through. Opened a new question on that
